src/app/profile/settings/connected-accounts/connected-accounts.component.html

```diff
@@ -35,8 +35,14 @@ <h2 class="card-pf-title">Connected Accounts</h2>
             <div class="col-xs-2">
               <img class="icon connected-account-icon" src="../../../../assets/images/openshift_icon.svg">
             </div>
-            <div class="account-text-box col-xs-10">
+            <div class="account-text-box col-xs-6">
```

Comment:
The earlier version was wrong layout wise since there were three columns in 12 column grid with `col-xs-2`, `col-xs-10` and `col-xs-4`. Also, it had some affects on tooltip positioning.

Reply:
Ah I see. Thanks.